CWP/papers/HSF-CWP-2017-07_detsim/latex/DetectorSimulation.tex

```diff
@@ -0,0 +1,3473 @@
+contiguity.
+
+New areas of commonality across experiments need to be explored, for
+example in the domains of Fast Simulation and geometry. A great deal of
+common code is relied upon by many users for the benefit of the whole
+HEP community for the long term. It will be important to identify these
+pieces of code, remove the experiment dependency where appropriate, and
+ensure that the pieces that cannot be removed are either going to be
+maintained by a group that feels responsibility for them or will not
+become a bottleneck for any detector simulation in the future.
+
+The large reliance on simulations implies the need for careful
+assessment of the systematic uncertainties associated to Geant4
+predictions. New capabilities and tools are being developed to allow for
+the calculation of systematic uncertainties on the physics observables
+predicted by the Geant4 beam and detector simulations. In other words,
+kinematic distributions, efficiencies, and calibration constants will be
+predicted with systematic uncertainties resulting from the propagation
+of the uncertainties associated with the individual Geant4 physics
+models.
+
+There are specific challenges associated with the Intensity Frontier
+experimental programme. Properly simulating a neutrino experiment like
+NOvA ~\cite{Adamson:2016tbq}, MicroBooNE~\cite{MicroBooNE}, 
+MINERvA~\cite{Aliaga:2013uqz}, or DUNE ~\cite{Acciarri:2015uup}
+involves a three-part software stack, the first and last of which are
+relevant to this paper; the second, event generators such as GENIE, 
+is discussed elsewhere. The first element of the stack concerns
+the beamline and the neutrino flux prediction. Estimating the neutrino
+flux is a notorious problem because as weakly interacting particles,
+neutrinos offer no independent mechanisms for measuring the flux. A
+central problem is in the difficulty of accurately simulating the
+hadronic physics of meson production, especially in a thick target.
+Additionally, the simulated neutrino flux depends very strongly on
+minute details of the beamline, meaning that neutrino beam kinematic
+distributions must be described in great detail. Geant4 is the most
+commonly used software toolkit for this work, although FLUKA ~\cite{Ferrari:2005zk} 
+and MARS ~\cite{MARS} are also used. The final element of the stack is the
+detector simulation. Again there are numerous challenges in terms of
+efficiently simulating large numbers of events inside complex detector
+geometries and in handling subtle physics effects in the traversal of
+radiation across matter. Neutrino experiments in particular rely heavily
+on detector simulations to reconstruct neutrino energy, which requires
+accurate modeling of energy deposition by a variety of particles across
+a range of energies. Geant4 is the universal solution at this stage.
+
+Muon experiments such as Muon g-2 and Mu2e also face large simulation
+challenges, being Geant4 the primary simulation toolkit for
```

Comment:
Phrasing is pretty confusing. ", as Geant 4 is the primary..." for me scans a lot better.